source/includes/platform-limitations.rst

```diff
@@ -0,0 +1,9 @@
+Starting in ``mongosh`` 2.0:
```

Comment:
(reuse)
"Starting in mongosh 2.0" makes sense in the installation section, for the most part, but not in the release notes. It's weird in the release notes because it goes directly under the v2.0.0 heading. What about tweaking the language a little so that phrase is outside the include?